hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-update-jenkins-jobs.yaml

```diff
@@ -4,7 +4,10 @@
     triggers:
         - timed: 'H/15 * * * *'
     builders:
-        - shell: 'curl -fsS https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/update-jobs.sh | /bin/bash -'
+        - shell: |
+            curl -fsS https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/update-jobs.sh > update-jobs.sh
+            chmod +x update-jobs.sh
+            ./update-jobs.sh "hack/jenkins/job-configs:hack/jenkins/job-configs/kubernetes-jenkins"
```

Comment:
ah, OK. at first this reminded me of a docker mount, which is why I was confused. not sure if there's a way to make that more obvious.